storage/tmpfs.md

```diff
@@ -6,24 +6,30 @@ redirect_from:
 - /engine/admin/volumes/tmpfs/
 ---
 
-[Volumes](volumes.md) and [bind mounts](bind-mounts.md) are mounted into the
-container's filesystem by default, and their contents are stored on the host
-machine.
-
-There may be cases where you do not want to store a container's data on the host
-machine, but you also don't want to write the data into the container's writable
-layer, for performance or security reasons, or if the data relates to
-non-persistent application state. An example might be a temporary one-time
-password that the container's application creates and uses as-needed.
-
-To give the container access to the data without writing it anywhere
-permanently, you can use a `tmpfs` mount, which is only stored in the host
-machine's memory (or swap, if memory is low). When the container stops, the
-`tmpfs` mount is removed. If a container is committed, the `tmpfs` mount is not
-saved.
+[Volumes](volumes.md) and [bind mounts](bind-mounts.md) allow you to share files
+between the host machine and container. This allows you to persist data even
+after the container is stopped.
+
+If you're running Docker on Linux, you have a third option: `tmpfs` mounts.
+When you create a container with a `tmpfs` mount, the container has a way to
+create files outside the container's writable layer, just like it happens with
+volumes and bind mounts.
```

Comment:
This is strange: "just like it happens with ...". Maybe, 

> When you create a container with a `tmpfs` mount, the container can create files outside the container's writable layer (as it can with volumes and bind mounts).